src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -71,6 +82,11 @@
   private final AppContext appContext;
   private final ConfigOverridesApplier configOverridesApplier;
 
+  public static final String BATCH_METRICS_ENABLED_CONF_FILE = "batch_metrics_enabled.conf";
+  private boolean batchMetricsEnabled;
+  private boolean defaultBatchMetricsEnabled = false;
```

Comment:
We should keep it as-is to remain consistent with rcaEnabledDefaultValue. Additinally, this value might change, so we shouldn't make it final.

Reply:
Can you explain how can a default change ?

Reply:
When I mentioned "this value might change", I meant that we may choose to make it true by default. But yeah, since this comment, my notion of constants has changed. Will change it to be private static final.